### PR TITLE
added link to the Computer Science Ontology

### PR DESCRIPTION
**Pull request by angelosalatino**
We are releasing a new resource:

- Computer Science Ontology
- Web portal for crowdsourcing it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 *~
 *.Rproj.user
 *.Rproj
+cso/.DS_Store
+.DS_Store
```
Comment (stain) [inline .gitignore]: We definitely do not want `.DS_Store`, but only the last of these lines should be needed :)

**Comment by angelosalatino** [comment]
I fixed the gitignore as you suggested

**Comment by angelosalatino** [comment]
If someone reviews this, it would be very much appreciated. We have a deadline and we don't want to risk that people use a not existing link, we already provided, as effect of this pull request. 

Many thanks in advance

**Comment by dgarijo** [comment]
No worries, I usually do this as soon as I see it. I hadn't realized your PR was pending.

**Pull request merged**